sunpy/net/_attrs.py

```diff
@@ -243,3 +243,13 @@ class Physobs(SimpleAttr):
     registry can be found at
     https://sdac.virtualsolar.org/cgi/show_details?keyword=PHYSOBS.
     """
+
+
+class Source(SimpleAttr):
```

Comment:
Are these not defined under the VSO namespace?

Reply:
Moved from vso to net